public/stylesheet.css

```diff
@@ -931,3 +847,288 @@ body.order-by-relevance .grid-item {
   height: 0;
   width: 0;
 }
+
+@media (hover: hover) {
+  .grid-item__link:hover {
+    opacity: 1;
+    text-decoration: underline;
+  }
+
+  .copy-button:not(:disabled):hover::before {
+    cursor: pointer;
+    display: block;
+    z-index: 1;
+  }
+}
```

Comment:
Why do you've moved all the `@media` to the end? I think that this confuses more than clarifies.